### PR TITLE
93129-panel-styles

### PR DESCRIPTION
**Pull request by jtrinklein**
93129
### Review
 - [x] @boskya
 - [x] @joebri 

### After Merge

- [ ] [site](https://daptiv.github.io/style-guide) is updated

### CODE REVIEW

File: styles/_panel.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
+$panel-title-height: 38px;
```
Comment (boskya) [inline styles/_panel.scss]: As we discussed, pull this out into own file and structure

**Comment by jtrinklein** [comment]
i'm reworking the whole thing to split each partial into 
```
partialfile/_variables.scss
partialfile/_mixins.scss
partialfile/_styles.scss
_partialfile.scss
```

where `_partialfile.scss` just includes the `_variables` `_mixins` and `_styles` in the `partialfile` folder

File: docs/docs.scss [hunk 2]
```diff
@@ -13,6 +13,16 @@ $snippet-padding: 12px;
...
 $snippet-border: 2px solid #CCC;
 
+.icon-block {
+    width: 75px;
+    height: 75px;
+    display: inline-block;
+    background-color: #eee;
+    line-height: 75px;
+    text-align: center;
+    border-radius: 3px;
```
Comment (boskya) [inline docs/docs.scss]: nit: probably good idea to stick with alphabetical ordering of properties. I know this in docs and hence not covered by our standards but probably just a good idea to be consistent.

File: styles/dropdown/_styles.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.daptiv-dropdown {
```
Comment (boskya) [inline styles/dropdown/_styles.scss]: nice work pulling this out :+1:

File: styles/nav/_styles.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        &.disabled {
+            color: $color-medium-dark-gray;
+            cursor: default;
+
+            &:hover {
+                color: $color-medium-dark-gray;
+            }
+        }
+
+        &.menu {
```
Comment (boskya) [inline styles/nav/_styles.scss]: ideal: this could be on separate thing outside of nav, subnav.,it really is just a button with a dropdown arrow.

**Comment by boskya** [comment]
looks really good. Good work on this! :shipit:

File: styles/nav/_styles.scss (hunk 4 above)
Comment (jtrinklein) [inline styles/nav/_styles.scss]: are you thinking about adding it to iconography?

File: styles/_side-menu.scss [hunk 5]
```diff
@@ -16,6 +16,11 @@ $side-menu-item-horizontal-padding: 10px;
     padding: $side-menu-item-vertical-padding $side-menu-item-horizontal-padding;
 
     &:hover {
+        text-decoration: underline;
```
Comment (joebri) [inline styles/_side-menu.scss]: We don't underline regular menu items - is there a reason for doing so here?

File: styles/_variables.scss [hunk 6]
```diff
@@ -7,3 +7,7 @@ $icon-size-small: 16px;
 /* whitespace units */
 $standard-whitespace-unit: 8px;
 $half-whitespace-unit: $standard-whitespace-unit / 2;
+
+/* layout */
+$standard-gutter-width: 10px;
+$standard-line-height: 28px;
```
Comment (joebri) [inline styles/_variables.scss]: 28 seems like a large 'standard' line-height?

File: styles/nav/_styles.scss (hunk 4 above)
Comment (joebri) [inline styles/nav/_styles.scss]: @boskya not sure I agree since different parts of the app might have different requirements for 'menus' (??)

Comment (boskya) [inline styles/nav/_styles.scss]: @joebri When they do it is easy enough to add the menu under the containing component that is dictating a different style.

File: styles/_side-menu.scss (hunk 5 above)
Comment (jtrinklein) [inline styles/_side-menu.scss]: nope. I just like it and we don't have any styles for it yet officially

File: styles/nav/_styles.scss (hunk 4 above)
Comment (jtrinklein) [inline styles/nav/_styles.scss]: also, this isn't the dropdown menu, this is just the styling for the item which indicates a menu is available (menu tab in this case specifically) which is why it was moved to iconography. Here it was actually redefining the `icon` class and also defining a `menu` type of icon class. All of which pointed to it being an icon style and not a nav dependent style.

File: styles/_variables.scss (hunk 6 above)
Comment (jtrinklein) [inline styles/_variables.scss]: agreed, removed

**Pull request merged**